`static/gen_manifests.go` grows unboundedly

Say you do few build rounds for k0s, the file `static/gen_manifests.go` seems to grow fast and unbounded:

```
jussi@ip-172-31-36-24:~/go/src/github.com/k0sproject/k0s$ touch hack/tools/Makefile.variables && make static/gen_manifests.go && ls -lah static/gen_manifests.go 
CGO_ENABLED=0 docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go install github.com/kevinburke/go-bindata/go-bindata@v3.23.0+incompatible
docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
-rw-r--r-- 1 jussi jussi 1.7M Sep 20 12:05 static/gen_manifests.go
jussi@ip-172-31-36-24:~/go/src/github.com/k0sproject/k0s$ touch hack/tools/Makefile.variables && make static/gen_manifests.go && ls -lah static/gen_manifests.go 
CGO_ENABLED=0 docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go install github.com/kevinburke/go-bindata/go-bindata@v3.23.0+incompatible
docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
-rw-r--r-- 1 jussi jussi 2.4M Sep 20 12:05 static/gen_manifests.go
jussi@ip-172-31-36-24:~/go/src/github.com/k0sproject/k0s$ touch hack/tools/Makefile.variables && make static/gen_manifests.go && ls -lah static/gen_manifests.go 
CGO_ENABLED=0 docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go install github.com/kevinburke/go-bindata/go-bindata@v3.23.0+incompatible
docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
-rw-r--r-- 1 jussi jussi 3.5M Sep 20 12:05 static/gen_manifests.go
```

Eventually this can lead to OOM issues during build:
```
CGO_ENABLED=0 docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go install github.com/kevinburke/go-bindata/go-bindata@v3.23.0+incompatible
docker run --rm -v '/home/jussi/go/src/github.com/k0sproject/k0s/build/cache':/run/k0s-build -v '/home/jussi/go/src/github.com/k0sproject/k0s':/go/src/github.com/k0sproject/k0s -w /go/src/github.com/k0sproject/k0s -e GOOS -e CGO_ENABLED -e GOARCH --user 1000:1001 k0sbuild.docker-image.k0s go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
Makefile:159: recipe for target 'static/gen_manifests.go' failed
make: *** [static/gen_manifests.go] Error 137
```

In case of gopls in use (vscode etc. envs) this can also lead to very high MEM usage for it as it tries to "index" and parse the file.

--- a/Makefile
+++ b/Makefile
@@ -156,6 +156,7 @@ pkg/apis/%/.client-gen.stamp: .k0sbuild.docker-image.k0s hack/tools/boilerplate.
 codegen_targets += static/gen_manifests.go
 static/gen_manifests.go: .k0sbuild.docker-image.k0s hack/tools/Makefile.variables
 static/gen_manifests.go: $(shell find static/manifests -type f)
+	-rm -f -- '$@'
 	CGO_ENABLED=0 $(GO) install github.com/kevinburke/go-bindata/go-bindata@v$(go-bindata_version)
 	$(GO_ENV) go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
 

